src/Workspaces/Core/Portable/Shared/Utilities/SymbolEquivalenceComparer.ParameterSymbolEqualityComparer.cs

```diff
@@ -78,5 +81,12 @@ public int GetHashCode(IParameterSymbol x)
                     _symbolEqualityComparer.SignatureTypeEquivalenceComparer.GetHashCode(x.Type));
             }
         }
+
+        public static bool AreEquivalent(RefKind rk1, RefKind rk2, bool distinguishRefFromOut)
+        {
+            return distinguishRefFromOut
+                ? rk1 == rk2
+                : (rk1 == RefKind.None) == (rk2 == RefKind.None);
```

Comment:
There's already that .IsRefOrOut helper being used just a few lines prior. Please use for consistency's sake.